govcd/edgegateway.go

```diff
@@ -634,3 +634,23 @@ func (eGW *EdgeGateway) RemoveIpsecVPN() (Task, error) {
 	}
 	return eGW.AddIpsecVPN(ipsecVPNConfig)
 }
+
+// getProxiedEdgeEndpoint helps to get root endpoint for Edge Gateway using the
+// NSX API Proxy and can optionally append suffix which must have it's own leading /
+func (eGW *EdgeGateway) getProxiedEdgeEndpoint(append string) (string, error) {
```

Comment:
I am confused by having a `get` (which suggests a query) and `append` ( which I associate with "modify") in the same signature. If the purpose is to return an item with a suffix, I suggest changing `append` into `optionalSuffix`

Reply:
Hopefully the current naming is better.